contracts/StakeRegistry.sol

```diff
@@ -159,6 +159,10 @@ contract StakeRegistry is IStakeRegistry, Ownable, ReentrancyGuard {
         allowedTokens.push(_dslaTokenAddress);
     }
 
+    function owner() public view override(IStakeRegistry, Ownable) returns (address) {
+        return super.owner();
+    }
```

Comment:
Is that really necessary?

What is preventing us from doing _stakeRegistry.owner() without this?

The protocol already owns the registry, so I'm assuming there is no need to call super.owner() since we haven't overridden owner in this contract.